pkg/core/resources/store/options.go

```diff
@@ -158,6 +158,7 @@ type ListOptions struct {
 	PageOffset string
 	FilterFunc ListFilterFunc
 	NamePrefix string
+	Ordered    bool
```

Comment:
You are adding this but there's 0 actual usage of it...

Reply:
It's used in pagination store https://github.com/kumahq/kuma/pull/5391/files#diff-e245a0b6bb1473059fa913b2e2f07c2a0773583a0738f403749eb5a1630027b1R49